bootstrap/kubeadm/controllers/kubeadmconfig_controller.go

```diff
@@ -44,11 +50,6 @@ import (
 	"sigs.k8s.io/cluster-api/util"
 	"sigs.k8s.io/cluster-api/util/patch"
 	"sigs.k8s.io/cluster-api/util/secret"
-	ctrl "sigs.k8s.io/controller-runtime"
-	"sigs.k8s.io/controller-runtime/pkg/client"
-	"sigs.k8s.io/controller-runtime/pkg/controller"
-	"sigs.k8s.io/controller-runtime/pkg/handler"
-	"sigs.k8s.io/controller-runtime/pkg/source"
```

Comment:
We're not using local goimports in this repository

Reply:
editor auto modification, will be happy to patch up once as part of a final commit/squash once everything else is sorted out.